schism/main.c

```diff
@@ -374,6 +374,23 @@ static void key_event_reset(struct key_event *kk, int start_x, int start_y)
 
 /* -------------------------------------------- */
 
+static int is_modifier_key(schism_keysym_t sym)
```

Comment:
IMO this would be better moved into keyboard.c as `kbd_key_is_modifier`

Reply:
That's a good idea :-)